notebooks/data_quality.ipynb

```diff
@@ -0,0 +1,1833 @@
+{
```

Comment:
<p>Maybe also comment here that the DL1 datacheck files should have been "reprocess" using the cherenkov transparency script</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I plan to copy the (updated) older files to the standard location, and then we will have the cherenov transparency script as part of the onsite automatic processing, so I think it is not necessary.